src/v/kafka/server/handlers/topics/types.cc

```diff
@@ -185,6 +185,10 @@ to_cluster_type(const creatable_topic& t) {
       = get_tristate_value<std::chrono::milliseconds>(
         config_entries, topic_property_retention_local_target_ms);
 
+    cfg.properties.remote_delete
```

Comment:
can this change be squashed with handling for the property in create topics request ?